src/main/java/edu/hm/hafner/analysis/parser/TrivyParser.java

```diff
@@ -23,8 +23,8 @@
  */
 public class TrivyParser extends JsonIssueParser {
     private static final String VALUE_NOT_SET = "-";
-    private static final String TRIVY_VULNERABILITY_LEVEL_TAG_CRITICAL = "critcal";
-    private static final String TRIVY_VULNERABILITY_LEVEL_TAG_HIGH = "high";
+    private static final String TRIVY_VULNERABILITY_LEVEL_TAG_CRITICAL = "critical";
+    //private static final String TRIVY_VULNERABILITY_LEVEL_TAG_HIGH = "high";
```

Comment:
Delete
```suggestion
```

Reply:
I've tweaked it a bit to remove the CRITICAL instead.

After reading the other Thomas' comment I realized it's probably better to default to Error if the severity isn't known (i.e. always assume the worst), however it's probably a moot point since I don't imagine it's possible to omit the severity in the XML being parsed.